doc/api/cli.md

```diff
@@ -156,6 +156,40 @@ Currently the support for run-time snapshot is experimental in that:
    a report in the [Node.js issue tracker][] and link to it in the
    [tracking issue for user-land snapshots][].
 
+### `--compile=path`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+> This feature is currently being designed and will still change.
+
+Compiles a JavaScript file into a single executable application.
```

Comment:
I feel that SEAs should be explained in the currently added doc instead of providing an external link. PTAL and let me know if there are some things that can be clarified.